### PR TITLE
[Ready for Review] Bahasa Indonesia - Meet 2023 Nov 20 update and fix term consistency

### LINKED ISSUE #132: Bahasa Indonesia - Introduction to Web Accessibility
I would like to update the Bahasa Indonesia translation to meet update on 2023 Nov 20 of  the following resource:
- Page URL:https://www.w3.org/WAI/fundamentals/accessibility-intro

I have read the information on Translating WAI Documents at https://www.w3.org/WAI/about/translating/

I will wait for confirmation that the resource is ready for translation.

### PR DESCRIPTION
**Pull request by frirasyidi**
The following changes includes:

- translation update to meet changes made on the English version up to 2023 Nov 20
- changes on terms used in the translation to improve consistency
- changes on sentences phrasing and choice of words to improve comprehension when spoken

### CODE REVIEW

**Comment by frirasyidi** [comment]
Note regarding a term changes:

> -   perceive, understand, navigate, and interact with the Web

- "perceive", is changed from "mepersepsikan" (percepting) to "**menerima** konten" (receiving content) as there is no equivalent in Bahasa Indonesia for the word "perceive".
- arguably, "**mengonsumsi** konten" (consuming content) is a bit closer to what's probably being referred to here, but as I am trying to stick to the exact English phrasing, I am connecting the translation to the concept of "perceivable" which in translation is closer to "dapat **diterima** indra" (can be perceived by senses). Another "perceivable" equivalence would be "dapat dirasakan" (can be felt) but it infers feelings/emotions rather than senses (sight, hearing, touch).
- despite "perceptable" means "jelas" (clearly perceived), "persepsi" in Bahasa Indonesia refers more to "a subject's perspective" which is far from being "clear" and prone to different interpretation depending on the subject.
- changes also made based on a discussion with other accessibility practitioner in Indonesia which agrees to the term being translated closer to "dapat diterima indra".

**Comment by frirasyidi** [comment]
@remibetin this translation is ready for review. I'm adding notes above so that the reviewer can understand the reason behind a particular changes in term.

**Comment by remibetin** [comment]
This PR is ready for review! 🚀

File: content/index.id.md [hunk 1]
```diff
@@ -76,102 +76,103 @@ Itulah mengapa dampak dari disabilitas berubah drastis pada Web, karena Web meng
...
+-   neurologis
 -   fisik
 -   bicara
 -   penglihatan
 
 Aksesibilitas Web juga memberikan manfaat bagi orang-orang yang *tidak* menyandang disabilitas, sebagai contoh:
 
 -   orang-orang yang menggunakan ponsel, jam tangan pintar, TV pintar, berbagai perangkat dengan layar kecil, mode input yang berbeda, dsb.
--   lansia dengan kemampuan yang terbatas karena usia
+-   orang tua dengan kemampuan yang terbatas karena usia
```
Comment (remibetin) [inline content/index.id.md]: Original text: 
> older people with changing abilities due to ageing

@frirasyidi My automatic translation tool translates "orang tua" into "parents" and "lansia" into "older people". That would make "lansia" closer to the original intent. What do you think?

**Review by remibetin — COMMENTED** [review]
@frirasyidi I have skimmed your translation and added a comment for you to consider.

Thanks again for your contribution!

File: content/index.id.md (hunk 1 above)
Comment (frirasyidi) [reply content/index.id.md]: Yeah, you are correct. I've changed it back to "lansia".

**Comment by remibetin** [comment]
Thank you @frirasyidi! 

Your translation is now ready to publish, and we will do so very soon.

**Comment by shawna-slh** [comment]
Thank you!

**Comment by remibetin** [comment]
QA done: ✅

**Comment by remibetin** [comment]
This translation update has been published today 🚀 Thank you @frirasyidi!

See https://www.w3.org/WAI/fundamentals/accessibility-intro/id

**Comment by frirasyidi** [comment]
Thanks you as well @remibetin!

**Pull request merged**